src/components/Layout/Layout.test.jsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+
+import { useAuth } from '../../providers/Auth';
+import Layout from './Layout.component';
+
+const mockHistoryPush = jest.fn();
+const mockLogout = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useHistory: () => ({
+    push: mockHistoryPush,
+  }),
+}));
+
+jest.mock('../../providers/Auth', () => ({
+  useAuth: jest.fn(() => ({ authenticated: false })),
+}));
+
+describe('Layout', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  it('should renders', () => {
+    render(
+      <Layout>
+        <div id="content">Content</div>
+      </Layout>
+    );
+
+    const container = document.querySelector('#content');
+    expect(container).toBeTruthy();
```

Comment:
why not using a `data-test-id` in order to use a `getByTestId()` instead?
actually, why not using any other HTML element that can be reachable by role using react-testing-library?